assets/element-templates.css

```
@@ -1,14 +1,24 @@
+.bio-properties-panel {
+  --select-template-background-color: var(--add-entry-hover-background-color);
+  --select-template-hover-background-color: var(--color-blue-205-100-45);
+  --select-template-fill-color: var(--add-entry-hover-fill-color);
+  --select-template-label-color: var(--add-entry-label-color);
```

I am not sure whether we should re-use semantic names of different components.

```suggestion
  --select-template-background-color: var(--color-blue-205-100-50);
  --select-template-hover-background-color: var(--color-blue-205-100-45);
  --select-template-fill-color: var(--color-white);
  --select-template-label-color: var(--color-white);
```

---

Imagine we change the color for `add-entry`. Should it change for selected template buttons too?

---

Agree that we should it keep it independently 👍 I added the changes you suggested.